prov/rxm/src/rxm_cq.c

```diff
@@ -1391,12 +1392,16 @@ void rxm_ep_do_progress(struct util_ep *util_ep)
 			} else {
 				ret = 1;
 			}
+			if (OFI_UNLIKELY(!(--rxm_ep->cq_eq_fairness)))
```

Comment:
Please drop the OFI_UNLIKELY bit.  Honestly it's overused in the code and may not be correct given that the fairness value is user settable.  To be a little safer here, I would convert this check to <= 0

Reply:
Thanks, sounds good. Changed.